### PR TITLE
Add fuzzy search on name and location and add new search fields

### PR DESCRIPTION
**Pull request by Anakin100100**

### CODE REVIEW

File: db/migrate/20220212134157_add_area_fields_to_institution.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 class AddAreaFieldsToInstitution < ActiveRecord::Migration[7.0]
   def change
+<<<<<<< HEAD
```
Comment (micorix) [inline db/migrate/20220212134157_add_area_fields_to_institution.rb:3]: "Merge leftovers"

**Comment by micorix** [comment]
Please provide a PR name which describes added feature or place the description of added feature in PR desc. E.g. full-text search on name_query and area_name fields

**Review by micorix — CHANGES_REQUESTED** [review]

File: db/migrate/20220212134157_add_area_fields_to_institution.rb (hunk 1 above)
Comment (Anakin100100) [reply db/migrate/20220212134157_add_area_fields_to_institution.rb]: I don't know why my ide didn't detect that while all tests passed locally

**Comment by micorix** [comment]
@Anakin100100 do I need to run some command to apply these changes to the db?

**Comment by Anakin100100** [comment]
yes rails db:migrate, I'm going to update the docs with that information

**Comment by micorix** [comment]
I know we discussed (not)adding db regeneration script to the entrypoint, but how about migrations?

**Comment by Anakin100100** [comment]
running migrations should be part of the db regeneration and it will be really easy to add there provided that we regenerate the database every time a new production version is released. It would also play well with the regeneration script.

**Comment by micorix** [comment]
Ok,cool.

Let's merge it then (we're on tight schedule). 

What's the ETA of regeneration script?

**Comment by Anakin100100** [comment]
around 2 days, but that's going to be a separate pull request

**Review by micorix — APPROVED** [review]

**Pull request merged**